src/components/utdragFraSykefravaeret/UtdragFraSykefravaeret.tsx

```diff
@@ -76,6 +77,7 @@ export const SykmeldingTittelbeskrivelse = ({
           {periode}
           {graderinger}
         </div>
+        {erViktigInformasjon && <img alt="Mer" src={MerInformasjonImage} />}
```

Comment:
Hva viser egentlig `MerInformasjonImage`? Lurer uansett på om `"Mer informasjon"` kan være riktigere som `alt` property 😊
```suggestion
        {erViktigInformasjon && <img alt="Mer informasjon" src={MerInformasjonImage} />}
```

Reply:
Litt gammel moro her, men det er denne figuren med tre prikker:
<img width="173" alt="image" src="https://github.com/navikt/syfomodiaperson/assets/37441744/73a53afa-de7b-4a46-9729-999b692c8043">

Kan endre alt-teksten 👍🏼